### PR TITLE
BFD-897: Fix and diagnose unstable Jenkins builds

### PR DESCRIPTION
**Pull request by karlmdavis**
### Change Details

This PR fixes PipelineApplicationIT.noRifData test case.

It also adds more logging for the PipelineManagerIT test case that's failing.

### Acceptance Validation

If the build passes in GH Actions, let's push it.

### Feedback Requested

N/A

### External References

- [BFD-897](https://jira.cms.gov/browse/BFD-897)

### Security Implications

There are no security implications for this PR.

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

File: apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineManager.java [hunk 1]
```diff
@@ -39,14 +39,7 @@
 import org.slf4j.LoggerFactory;
 
 /** Orchestrates and manages the execution of {@link PipelineJob}s. */
-public final class PipelineManager {
```
Comment (jzulim) [inline apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineManager.java:42]: Dead code. 👍

File: apps/bfd-pipeline/bfd-pipeline-app/src/test/java/gov/cms/bfd/pipeline/app/PipelineManagerIT.java [hunk 2]
```diff
@@ -22,296 +22,352 @@
...
 public final class PipelineManagerIT {
   private static final Logger LOGGER = LoggerFactory.getLogger(PipelineManagerIT.class);
 
+  @Rule
+  public TestWatcher testCaseEntryExitLogger =
+      new TestWatcher() {
+        /** @see org.junit.rules.TestWatcher#starting(org.junit.runner.Description) */
+        @Override
+        protected void starting(Description description) {
+          LOGGER.info("{}: starting.", description.getDisplayName());
```
Comment (mackec1) [inline apps/bfd-pipeline/bfd-pipeline-app/src/test/java/gov/cms/bfd/pipeline/app/PipelineManagerIT.java:42]: Always love logging that stuff; altho I try to put about 100 * and BIGGEST ALL-CAPS I can find!

**Review by mackec1 — APPROVED** [review]
One step closer to nailing this!

**Review by jzulim — APPROVED** [review]
let it rip. let's see those logs. 🎭

**Pull request merged**